src/elements/play-preview.ts

```diff
@@ -169,4 +170,27 @@ export class PlayPreview extends LitElement {
         this.#meta
       )
   }
+
+  private mockImageUpload = (
```

Comment:
This doesn't use any class state. Please move to a standalone function below outside the class to clarify.

Here's my thinking: a method has access to the entire class state, a function does not. When I see a function, I know that I can read it totally independent of whatever is happening inside of a class. In Go, this is explicit:

```go
type State struct {
  counter int
}

// Why do I tell everyone I need class state when I don't? Use a function, not a method.
func (state * State) sayHello(name string) {
  fmt.Printf("hello %s\n", name)
}
```

It doesn't matter in this case but [dead code methods don't treeshake](https://esbuild.github.io/try/#YgAwLjIwLjIALS1idW5kbGUgLS1mb3JtYXQ9ZXNtIC0tdHJlZS1zaGFraW5nPXRydWUAZQBlbnRyeS50cwBleHBvcnQgY2xhc3MgQ2xhc3MgewogIHByaXZhdGUgc3RhdGljIHN0YXRpY01ldGhvZCgpIHt9CiAgcHJpdmF0ZSBtZXRob2QoKSB7fQp9CgpuZXcgQ2xhc3MoKQ).

```ts
export class Class {
  private static staticMethod() {} // Not called and not dropped.
  private method() {} // Not called and not dropped.
}

new Class()
```